src/lib/themes-test.js

```diff
@@ -33,6 +33,146 @@ describe("lib/themes", () => {
         expect(output).to.deep.equal(theme);
       });
     });
+    it("should migrate themes without popup/popup_text", () => {
```

Comment:
Nit, if you end up to force push / update this PR again, let's add an empty line between the `it(...)` calls (don't bother if you are not going to update it again for now).

Reply:
Added a few blank lines for consistency.